src/docs/backup.dox

```diff
@@ -98,29 +104,31 @@ and removing log files from the original database home:
 
 1. Perform a full backup of the database (as described above).
 
-2. Perform a full database checkpoint.
```

Comment:
If the user does not do a checkpoint then the log files cannot be archived.  So opening the backup cursor on the log files will continue to accumulate more and more files and the user will be copying them multiple times.  That was the reason that step is in there.